server.go

```diff
@@ -1978,3 +2029,22 @@ type channelzServer struct {
 func (c *channelzServer) ChannelzMetric() *channelz.ServerInternalMetric {
 	return c.s.channelzMetric()
 }
+
+// validateSendCompressor returns an error when given compressor name cannot be
+// handled by the server or the client based on the advertised compressors.
+func validateSendCompressor(name, clientCompressors string) error {
+	if name == "identity" {
+		return nil
+	}
```

Comment:
This makes me feel uneasy, but I'm not sure if there's a much better option.

At the least, this should be documented in SetSendCompressor, and we should probably go ahead and define a const in the encoding package for it, as well.

Reply:
My bad, there is a const for identity in encoding pkg. Updated the documentation to include:
```
// It must be noted that compressor name "identity" disables the outbound compression.
```